docs/systemd.md

```diff
@@ -58,11 +58,22 @@ dinamically, you can find the remaining options needed in
 
 ## Systemd units on WSL
 
-Unfortunately, `systemd` is not officially supported on WSL.
+`systemd` is supported on WSL starting from version `0.67.6` and higher. Note that you need to have Windows 11 to install this version.
 
-However, there are several unofficial workarounds for it and some have been known to work with `tezos-packaging`'s units.
+To enable `systemd` startup on boot you need to add the following lines to `/etc/wsl.conf` file:
```

Comment:
Am I right that one should do so _after_ installing wsl with Ubuntu in the conventional way?

If so, I think it's worth specifying this and also adding a command for how to do the edit, probably using `nano` which should be installed by default